autogpt_platform/backend/backend/integrations/providers.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class ProviderName(str, Enum):
+    GITHUB = "github"
+    GOOGLE = "google"
+    NOTION = "notion"
```

Comment:
Add this to docs as another place that needs updated when adding a provider

Reply:
what is this even? its not all our providers lol. Is this oauth login providers/]?

Reply:
Removed in this PR, properly addressed in #8725